<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HighScoreDialog</class>
 <widget class="QDialog" name="HighScoreDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="player4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Text label</string>
   </property>
  </widget>
  <widget class="QLabel" name="score1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="score5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="player2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Text label</string>
   </property>
  </widget>
  <widget class="QLabel" name="score3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="player1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Text label</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="score2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="score4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="player3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Text label</string>
   </property>
  </widget>
  <widget class="QLabel" name="player5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Text label</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
